src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT + 1);
+        double radius = random.nextDouble() * 100;
+        double firsLeg = random.nextDouble() * 100;
+        double secondLeg = random.nextDouble() * 100;
+        String color = colorSupplier.getRandomColor();
+        double length = random.nextDouble() * 100;
+        double width = random.nextDouble() * 100;
+        double base = random.nextDouble() * 100;
+        double top = random.nextDouble() * 100;
+        double height = random.nextDouble() * 100;
+        double side = random.nextDouble() * 100;
+        if (figureNumber == 1) {
+            Circle circle = new Circle(radius, color);
+            return circle;
```

Comment:
lets create private methods without params that create each random figure and invoke those methods in switch case
like this `getRandomCircle()`